certify-mock-mdl.properties

```diff
@@ -0,0 +1,94 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.mock.integration
+mosip.certify.integration.audit-plugin=LoggerAuditService
+mosip.certify.integration.vci-plugin=MDocMockVCIssuancePlugin
+mosip.certify.plugin-mode=VCIssuance
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.cache.secure.individual-id=false
+mosip.certify.cache.store.individual-id=true
+mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
+mosip.certify.mock.vciplugin.mdoc.issuer-key-cert=${mosip.certify.mock.vciplugin.mdoc.issuer-key-cert}
```

Comment:
why do we have both key and value same ?

Reply:
Changed the placeholder value as well to reflect the property key change
@vishwa-vyom Can we keep the placeholder as `mosip.certify.mock.vciplugin.mdoc.issuer-crypto-details` or as previous `mosip.certify.mock.vciplugin.issuer.cert`

Reply:
Modifying the value to mosip.certify.mock.mdoc.issuer-key-cert